nodejs/week3/MainHomework/src/backend/api/meal_07_maxPricebyQuery.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const pool = require("../database");
+
+router.get("/", (request, response) => {
+
+  console.log('maxPrice')
+  response.send('maxPrice')
+  // const maxPrice = request.query.maxPrice;
+
+  // const queryText = `SELECT * FROM meal WHERE pice <'${maxPrice}'`
```

Comment:
// const queryText = `SELECT * FROM meal WHERE **price** <'${maxPrice}'`
why you commented out?